packages/mdc-floating-label/_mixins.scss

```diff
@@ -21,60 +21,149 @@
 //
 
 @import "@material/rtl/mixins";
+@import "@material/theme/variables";
 @import "@material/theme/mixins";
+@import "@material/feature-targeting/functions";
+@import "@material/feature-targeting/mixins";
+@import "@material/animation/variables";
+@import "@material/typography/mixins";
 @import "./variables";
 
-@mixin mdc-floating-label-ink-color($color) {
-  @include mdc-theme-prop(color, $color);
-}
+@mixin mdc-floating-label-core-styles($query: mdc-feature-all()) {
+  $feat-structure: mdc-feature-create-target($query, structure);
+  $feat-animation: mdc-feature-create-target($query, animation);
 
-// Used for textarea in case of scrolling
-@mixin mdc-floating-label-fill-color($color) {
-  @include mdc-theme-prop(background-color, $color);
-}
+  // postcss-bem-linter: define floating-label
+  .mdc-floating-label {
+    @include mdc-typography(subtitle1, $query: $query);
```

Comment:
nit: Please use `$exclude-props` to exclude `line-height` in typography to avoid duplication with below CSS override.

BUILD will throw an error otherwise. :)

`/* @alternative */` comment before CSS override would also fix this.

Reply:
Fixed.